Sources/TuistCore/Graph/GraphTraverser.swift

```diff
@@ -181,6 +181,19 @@ public class GraphTraverser: GraphTraversing {
             .first { $0.target.product == .appClip }
     }
 
+    public func buildsForMacCatalyst(path: AbsolutePath, name: String) -> Bool {
+        let dependenciesSupportMacCatalyst = allDependenciesSatisfy(from: .target(name: name, path: path)) { dependency in
+            if let target = self.target(from: dependency) {
+                return target.target.supportsCatalyst
+            } else {
+                // TODO: - Obtain this information from pre-compiled binaries
```

Comment:
We'll likely need this logic for other area related to multi-platform dependencies. Not sure what it should look like or where it should live, but wanted to call it out.

Reply:
I'll ping you once it's ready.